Editor/AGS.Editor/app.manifest

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
```

Comment:
Mmm, that's probably for the best, I don't know if there is some way we can automate updating version number however. I imagine doing it manually gets old really fast.